apps/wallet-mobile/src/features/Send/useCases/ListAmountsToSend/ListAmountsToSendScreen.tsx

```diff
@@ -48,6 +48,9 @@ export const ListAmountsToSendScreen = () => {
   )
 
   const onEdit = (tokenId: string) => {
+    const tokenInfo = tokenInfos.filter((tokenInfo) => tokenInfo.id === tokenId)[0]
```

Comment:
avoid index accessing whether `const [tokenInfo]` or update to `tokenInfos.find()`

```suggestion
    const tokenInfo = tokenInfos.find((tokenInfo) => tokenInfo.id === tokenId)
```

Reply:
@jemubm can you check and test it, it needs to be included into `4.11.0` otherwise the edit will crash.

Reply:
done thanks